exporter/datasetexporter/README.md

```diff
@@ -43,6 +43,17 @@ The process of populating the serverHost attribute works as follows:
 
 Make sure to provide the appropriate server host value in the `serverHost` attribute to ensure the proper functionality of DataSet and accurate handling of events.
 
+### Attributes
+
+If exporting of attributes of given type are enabled, then they are exported in following order:
+1. Body
+2. Resource attributes
+3. Scope attributes
+4. Log attributes
+
+In case of name conflict the later attribute name will be appended with the `export_distinguishing_suffix` value. If the value is empty string then the last value wins.
```

Comment:
If there is a name conflict, the `export_distinguishing_suffix` value is appended to the later attribute's name. If the the `export_distinguishing_suffix` value is an empty string, then the preceding attribute's name is used. 

I think you mean preceding here - the name of the attribute in the hierarchy before the attribute with the naming conflict and empty `export_distinguishing_suffix`?

Reply:
It's indeed confusing. I will add some examples :)

> If there is a name conflict, the export_distinguishing_suffix value is appended to the later attribute's name. If the the export_distinguishing_suffix value is an empty string, then the value from the last attribute is used.

Reply:
I have added this as separate line in the attributes section.